website/case-studies/implementation-of-flint-modules.md

```diff
@@ -0,0 +1,407 @@
+
+To determine the volume of turnover through time, a calculation similar to the one below is carried out:
+
+```
+Turnover= 1 - (1 - turnover_rate) ^ time
+```
+
+Within the FLINT, the maximum decay rate is set at 0.99 999 999 998, and the minimum decay rate is 0.000001. These restrictions are in place to ensure there are no invalid calculations (Example: dividing by zero) occurs in the system.
+
+## Decomposition Module
+
+Decomposition represents heterotrophic respiration of litter and soil organic matter. The process of modelling decomposition is akin to turnover in plant material, where a proportion of biomass in litter and soil organic matter decays through time. The rates of this will vary with location based on the prevailing environmental conditions.
+
+## RothC Soil Carbon Module
+
+RothC is a module used for simulating the turnover of soil carbon in non-waterlogged soils under different vegetation types. It is based on the decomposition of organic matter as it relates to mean temperature and moisture, and proportion of resistant and decomposable organic material. It runs on the monthly time step interval.
+
+It is better than the other modules because its input data is readily available and there is considerable overlap with hybrid growth modules and WOFOST modules. For a detailed explanation and better understanding you can refer to the [RothC guide](https://www.verdeterreprod.fr/wp-content/uploads/2020/05/RothC_guide_WIN.pdf).
+
+### Inputs:
+
+-   Time Step (Monthly).
+-   Pool 1: Input of plant residue.
+-   Pool 2: Input of farmyard manure.
+-   Pool 3: Total organic carbon.
+-   Pool 4: Microbial biomass carbon.
+-   Rainfall.
+-   Open pan evaporation.
+-   Mean air temperature.
+-   Clay content of soil.
+-   Decomposability of incoming plant material.
+-   Soil Cover.
+-   Depth of soil layer sampled.
+
+### Output:
+
+-   Total Organic Carbon.
+-   Microbial Biomass Carbon.
+
+|![Pictorial representation of RothC Module.](../static/img/RothC-module-case-study.png) |
+|:--:|
+| Pictorial representation of RothC Module. |
+
+## Chapman Richards Module
+
+The Chapman Richards module is a tier-3 module. It is used for the estimation of greenhouse gas emissions and removals due to changes in biomass carbon for forests using Chapman Richards equation. We can find aboveground and belowground biomass using this module.
+
+```
+AGC = * [ 1 -  e ^ { -k *Age } ] ^ { ( 1 / (1-m) }
+```
+
+-   **AGC**: Above-ground stand carbon (Tonnes carbon per hectare).
+-   **Max**: Asymptote – maximum peak carbon yield (Tonnes carbon per hectare).
+-   **k**: Parameter used in modelling tree growth (dimensionless).
+-   **Age**: Age of the forest (years).
+-   **m**: Parameter used in modelling tree growth (dimensionless).
```

Comment:
This is an excellent description - equations need to have all the parameters explained.